maps/aurora/aurora-4_mainlevel.dmm

```diff
@@ -49894,6 +49894,7 @@
 "bIJ" = (
 /obj/structure/cult/tome,
 /obj/item/clothing/under/suit_jacket/red,
+/obj/item/device/radio/headset/headset_service,
```

Comment:
Dont they have a locker where that can be placed in ?

Reply:
I don't mean to be snide, but have you read the description of this PR?

Because that exact question is answered within it.

Reply:
*Actually, no I havnt. I just looked at the code*